examples/hooks/App.tsx

```diff
@@ -15,6 +15,7 @@ import {
   InfiniteHits,
   Menu,
   Pagination,
+  RangeInput,
```

Comment:
The `<RangeInput>` doesn't display because the `<NumericMenu>` takes precedence over it. I think we can remove `<NumericMenu>` now.